src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder salaryInfo = new StringBuilder();
+        LocalDate startDate = LocalDate.parse(dateFrom);
+        LocalDate endDate = LocalDate.parse(dateTo);
```

Comment:
The date parsing here uses the default ISO_LOCAL_DATE format (yyyy-MM-dd). If the input dates are in a different format, such as dd-MM-yyyy, you should use the DateTimeFormatter defined on line 11 to parse the dates correctly.